src/chainparams.cpp

```diff
@@ -207,7 +196,7 @@ class CTestNetParams : public CMainParams {
 
         consensus.prePowLimit = consensus.powLimit;
         assert(maxUint/UintToArith256(consensus.powLimit) >= consensus.nPowAveragingWindow);
-        consensus.fPowAllowMinDifficultyBlocks = true;
+        consensus.fPowAllowMinDifficultyBlocks = false;
```

Comment:
Why change this to false?

It's `true` in Bitcoin 

https://github.com/bitcoin/bitcoin/blob/master/src/chainparams.cpp#L189